Unused '@ts-expect-error' directive

The latest sst (3.17.19) has a typecheck error:

.sst/platform/src/components/aws/remix.ts:481:13 - error TS2578: Unused '@ts-expect-error' directive.

`"sst": "3.17.21",`
```
.sst/platform/src/components/aws/remix.ts:481:13 - error TS2578: Unused '@ts-expect-error' directive.

481             // @ts-expect-error
                ~~~~~~~~~~~~~~~~~~~


Found 1 error in .sst/platform/src/components/aws/remix.ts:481
```

--- a/platform/src/components/aws/remix.ts
+++ b/platform/src/components/aws/remix.ts
@@ -478,7 +478,7 @@ export class Remix extends SsrSite {
           if (!file) return;
 
           try {
-            // @ts-expect-error
+            // @ts-ignore
             const vite = await import("vite");
             const config = await vite.loadConfigFromFile(
               { command: "build", mode: "production" },

